frontend/cypress/e2e/action_intervals.cy.js

```diff
@@ -11,50 +11,53 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/* global cy, describe, beforeEach, it */
```

Comment:
This line appears in multiple files but I'm not sure if it should be included in the comment.